code/modules/busy_space/organizations.dm

```diff
@@ -0,0 +1,461 @@
+		)
+
+/datum/lore/organization/tsc/vey_med
+	name = "Vey Medical"
+	short_name = "Vey Med"
+	acronym = "VM"
+	desc = "Vey-Med is one of the newer TSCs on the block and is notable for being largely owned and operated by Skrell. \
+	Despite the suspicion and prejudice leveled at them for their alien origin, Vey-Med has obtained market dominance in \
+	the sale of medical equipment-- from surgical tools to large medical devices to the Odysseus trauma response mecha \
+	and everything in between. Their equipment tends to be top-of-the-line, most obviously shown by their incredibly \
+	human-like FBP designs. Vey's rise to stardom came from their introduction of ressurective cloning, although in \
+	recent years they've been forced to diversify as their patents expired and Nanotrasen-made medications became \
+	essential to modern cloning."
+	history = ""
+	work = "medical equipment supplier"
+	headquarters = ""
+	motto = ""
+
+	ship_prefixes = list("VTV" = "transportation", "VMV" = "medical resupply")
+	destination_names = list(
+	"a outpost on Jargon 4"
+	)
+
+/datum/lore/organization/tsc/zeng_hu
+	name = "Zeng-Hu pharmaceuticals"
+	short_name = "Zeng-Hu"
+	acronym = "ZH"
+	desc = "Zeng-Hu is an old TSC, based in the Sol system. Until the discovery of Plasma, Zeng-Hu maintained a stranglehold \
+	on the market for medications, and many household names are patentted by Zeng-Hu-- Bicaridyne, Dylovene, Tricordrizine, \
+	and Dexalin all came from a Zeng-Hu medical laboratory. Zeng-Hu's fortunes have been in decline as Nanotrasen's near monopoly \
+	on plasma research cuts into their R&D and Vey-Med's superior medical equipment effectively decimated their own equipment \
+	interests. The three-way rivalry between these companies for dominance in the medical field is well-known and a matter of \
+	constant economic speculation."
+	history = ""
+	work = "pharmaceuticals company"
+	headquarters = ""
+	motto = ""
+
+	ship_prefixes = list("ZTV" = "transportation", "ZMV" = "medical resupply")
+	destination_names = list()
+
+/datum/lore/organization/tsc/ward_takahashi
+	name = "Ward-Takahashi General Manufacturing Conglomerate"
+	short_name = "Ward-Takahashi"
+	acronym = "WT"
+	desc = "Ward-Takahashi focuses on the sale of small consumer electronics, with its computers, communicators, \
+	and even mid-class automobiles a fixture of many households. Less famously, Ward-Takahashi also supplies most \
+	of the AI cores on which vital control systems are mounted, and it is this branch of their industry that has \
+	led to their tertiary interest in the development and sale of high-grade AI systems. Ward-Takahashi's economies \
+	of scale frequently steal market share from Nanotrasen's high-price products, leading to a bitter rivalry in the \
+	consumer electronics market."
+	history = ""
+	work = "electronics manufacturer"
+	headquarters = ""
+	motto = ""
+
+	ship_prefixes = list("WTV" = "freight")
+	destination_names = list(
+	"a station near Kelune"
+	)
+
```

Comment:
/datum/lore/organization/tsc/cybersun
	name = "Cybersun Industries"
	short_name = "Cybersun Ind."
	acronym = "CI"
	desc = "Cybersun Industries is a biotechnology company that primarily specializes on the research and development of human-enhancing augmentations. They are better known for their aggressive corporate tactics and are known to often subsidize pirate bands to commit acts of industrial sabotage. Cybersun Industries is often the target of conspiracy theorists due to their development of the first mindslave implant, as well as their open financing of, and involvement in, The Syndicate. They are one of Nanotrasen's largest detractors, and a direct competitor to BioTech Solutions."
	history = ""
	work = "RND company specializing in augmentations and implants."
	headquarters = "Luna"
	motto = ""

	ship_prefixes = list("CIND-T" = "transportation")
	destination_names = list()